py/picca/fitter2/chi2.py

```diff
@@ -379,10 +406,11 @@ def export(self):
         if hasattr(self, "minos_para"):
             g = f.create_group("minos")
             g.attrs['sigma'] = self.minos_para['sigma']
-            minos_results = self.best_fit.get_merrors()
+            minos_results = self.best_fit.merrors # self.best_fit.get_merrors()
```

Comment:
probably we can remove the comment, right? I don't think it is giving us any relevant information

Reply:
did just that